aten/src/ATen/native/quantized/library.cpp

```diff
@@ -82,13 +82,16 @@ TORCH_LIBRARY(quantized, m) {
   m.def(TORCH_SELECTIVE_SCHEMA("quantized::conv3d_unpack(__torch__.torch.classes.quantized.Conv3dPackedParamsBase packed_weights) -> (Tensor unpacked_weights, Tensor? B_origin)"));
   m.def(TORCH_SELECTIVE_SCHEMA("quantized::conv2d_stride(__torch__.torch.classes.quantized.Conv2dPackedParamsBase packed_weights) -> int[]"));
   m.def(TORCH_SELECTIVE_SCHEMA("quantized::conv2d_padding(__torch__.torch.classes.quantized.Conv2dPackedParamsBase packed_weights) -> int[]"));
+  m.def(TORCH_SELECTIVE_SCHEMA("quantized::conv2d_output_padding(__torch__.torch.classes.quantized.Conv2dPackedParamsBase packed_weights) -> int[]"));
```

Comment:
Not exactly sure where these missing quantized fn's were meant to be used, but I don't see any usage of them directly in pytorch source (specifically the 3 `quantized` namespace fn's that I added).

There were also 5 fn def's missing from the `_quantized` namespace, but judging from [this comment](https://github.com/pytorch/pytorch/blob/0c5cd8c2b9cdf473e30bbb1b49ca80ed442813df/aten/src/ATen/native/quantized/library.cpp#L165), the `_quantized` namespace has to do with mobile and will be removed soon.

Reply:
cc @jerryzh168 But adding the schemas for these seems like the right thing to do

Reply:
these are used when there are backends that does not implement the same interface for quantized::conv2d/quantized::conv_transpose_2d and need to extract parameters from packed_weights. e.g. glow.